SwiftLSPClient/Types/Basic.swift

```diff
@@ -83,6 +96,14 @@ public struct TextDocumentItem: Codable {
         self.version = version
         self.text = text
     }
+
+    public init(contentsAt path: String, version: Int = 1) throws {
```

Comment:
I've just noticed that String uses the init form `String(contentsOfFile: path)`. Probably best to match that convention, and rename this parameter `contentsOfFile`.